snapshot-type-azure/functions.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+DATETIME=`date +"%Y_%m_%d_%H_%M_%S"`
+
+make_backup () {
+
+    export FILENAME={{FILENAME}}
+    export CONTAINER={{CONTAINER}}
+    export ELASTICSEARCH_HOST={{ELASTICSEARCH_HOST}}
+    export ELASTICSEARCH_PORT={{ELASTICSEARCH_PORT}}
+    export DEBUG={{DEBUG}}
+    export BACKUP_REPOSITORY-{{BACKUP_REPOSITORY}}
```

Comment:
@darshanvarabhi Can you please rectify this Typo? It must be `export BACKUP_REPOSITORY`**=**`{{BACKUP_REPOSITORY}}` and not `export BACKUP_REPOSITORY`**-**`{{BACKUP_REPOSITORY}}`